client/components/data/query-active-theme/index.jsx

```diff
@@ -12,10 +12,7 @@ import { isRequestingActiveTheme } from 'state/themes/selectors';
 
 class QueryActiveTheme extends Component {
 	static propTypes = {
-		siteId: PropTypes.oneOfType( [
-			PropTypes.number,
-			PropTypes.oneOf( [ 'wpcom' ] )
```

Comment:
A theme can only be active on an actual site, not on all of WP.com 😄 

(Discovered while adding `wporg` to some other `propTypes` and `schema`s.)